utils/python/CIME/case.py

```diff
@@ -68,18 +68,20 @@ def get_value(self, item, attribute={}, resolved=True, subgroup=None):
         logging.info("Not able to retreive value for item '%s'" % item)
 
 
-    def get_resolved_value(self, item):
-        # TODO HACK - surely there is a better way?
```

Comment:
So this change is so that when there are recursive unresolved values as in for example EXEROOT=$SCRATCHROOT/$CASE/bld and SCRATCHROOT=/tmp/$USER  the search includes all of the xml files in case and not just the one pointed to by the current env_file object.